### PR TITLE
Adds some accessibility improvements

### PR DESCRIPTION
**Pull request by GlynnPhillips**
- Level 1 heading (visually hidden)
- Updating `id` so skip to content link works
- Adds level 2 heading for top stories (visually hidden)

### CODE REVIEW

File: views/front-page.html [hunk 1]
```diff
@@ -13,8 +13,10 @@
 {{/if}}
 
 <section
-	id="main-body"
+	id="site-content"
```
Comment (ironsidevsquincy) [inline views/front-page.html]: why this change

File: shared/components/section/section.js [hunk 2]
```diff
@@ -38,7 +38,7 @@ export default class extends Component {
 				{
 					cols.meta ?
 						<div data-o-grid-colspan={colspan(cols.meta)} className="section__column section__column--meta">
-							<SectionMeta title={this.props.title} />
+							<SectionMeta title={this.props.title} meta={this.props.cols.meta} />
```
Comment (ironsidevsquincy) [inline shared/components/section/section.js]: I would only pass in the things you need, in this case

```... isHidden={this.props.cols.meta.hide} ...```

File: views/front-page.html (hunk 1 above)
Comment (GlynnPhillips) [inline views/front-page.html]: This makes the skip links work, example where its failing on the homepage: https://next.ft.com/uk#site-content

and working on an article: https://next.ft.com/content/57981914-f588-11e5-96db-fc683b5e52db#site-content

I looked around and couldn't see the `id` being used else where

File: shared/components/section/section.js (hunk 2 above)
Comment (GlynnPhillips) [inline shared/components/section/section.js]: Sure, I will update this.

**Comment by ironsidevsquincy** [comment]
:+1:
